cg/meta/demultiplex/delete_demultiplex_api.py

```diff
@@ -52,7 +52,7 @@ def get_path_for_flow_cell(
 
     @property
     def status_db_presence(self) -> bool:
-        """Update about the presence of given flow cell in status_db"""
+        """Update about the presence of given flow cell in status_db."""
         return bool(self.status_db.get_flow_cell_by_name(flow_cell_name=self.flow_cell_name))
```

Comment:
I think the name of this property should be `flow_cell_exists_in_status_db` for readability. Also the a doc string like: `"""Returns wether a flow cell exists in status_db."""`

Reply:
fixed